queries/purescript/locals.scm

```diff
@@ -0,0 +1,4 @@
+(signature name: (variable)) @definition
+(function name: (variable)) @definition
+(pat_name (variable)) @definition
+(exp_name (variable)) @reference
```

Comment:
Should these not capture the variable node?

Reply:
Semantically it's something else. It's just that it appears in the tree under this name.

Reply:
All right. The function one should be `@definition.function`. I don't know what the other two are.